docs/UserManuals/DORA.md

```diff
@@ -76,13 +77,14 @@ In the next section, we'll demonstrate how to configure DevLake to implement DOR
 
 5. To make it simple, fields with a ![](https://i.imgur.com/rrLopFx.png) label are DORA-related configurations for every data source. Via these fields, you can define what are "incidents" and "deployments" for each data source. After all data connections have been configured, click `Next Step`
    - This team uses Jira issue types `Crash` and `Incident` as "incident", so choose the two types in field "incident". Jira issues in these two types will be transformed to "incidents" in DevLake.
-   - This team uses the GitHub action jobs named `deploy` and `build-and-deploy` to deploy, so type in `(?i)deploy` to match these jobs. These jobs will be transformed to "deployments" in DevLake.
-   ![](https://i.imgur.com/1JZA2xn.png)
+   - This team uses the GitHub action workflow named `Build workflow` deploy, so type in `(?i)Build` to match these workflows. Jobs in these workflow will be transformed to "deployments" in DevLake.
```

Comment:
I don't think this will solve the problem, as it can only tell DevLake jobs in which workflow runs can be collected, but it can't tell which jobs are `deployments`. Is it correct?